python/foglamp/tasks/north/sending_process.py

```diff
@@ -374,884 +279,654 @@ def __init__(self, loop=None):
+        # Command line parameter handling
+        self._log_performance, self._debug_level = handling_input_parameters()
+        _log_performance = self._log_performance
 
-        except Exception as ex:
-            message = _MESSAGES_LIST["e000017"].format(str(ex))
-            SendingProcess._logger.exception(message)
-            sys.exit(1)
         try:
-            self._storage_async = StorageClientAsync(self._mgt_address, self._mgt_port)
-            self._readings = ReadingsStorageClientAsync(self._mgt_address, self._mgt_port)
+            self._storage_async = StorageClientAsync(self._core_management_host, self._core_management_port)
+            self._readings = ReadingsStorageClientAsync(self._core_management_host, self._core_management_port)
             self._audit = AuditLogger(self._storage_async)
         except Exception as ex:
-            message = _MESSAGES_LIST["e000023"].format(str(ex))
-            SendingProcess._logger.exception(message)
+            SendingProcess._logger.exception(_MESSAGES_LIST["e000023"].format(str(ex)))
             sys.exit(1)
         else:
-            # Reconfigures the logger using the Stream ID to differentiates
-            # logging from different processes
             SendingProcess._logger.removeHandler(SendingProcess._logger.handle)
-            logger_name = _MODULE_NAME + "_" + str(self.input_stream_id)
-
-            SendingProcess._logger = logger.setup(logger_name, destination=_LOGGER_DESTINATION, level=_LOGGER_LEVEL)
-            
-            # TODO: Create a separate "MicroserviceManagementClient" like client for tasks
-            self._core_task_management_client = MicroserviceManagementClient(self._mgt_address, self._mgt_port)
+            logger_name = _MODULE_NAME + "_" + self._name
+            SendingProcess._logger = logger.setup(logger_name, level=logging.INFO if self._debug_level in [None, 0,
+                                                                                                           1] else logging.DEBUG)
+            _LOGGER = SendingProcess._logger
 
             try:
-                # Set the debug level
-                if self._log_debug_level == 1:
-                    SendingProcess._logger.setLevel(logging.INFO)
-                elif self._log_debug_level >= 2:
-                    SendingProcess._logger.setLevel(logging.DEBUG)
-
-                # Sets the reconfigured logger
-                _LOGGER = SendingProcess._logger
-
-                # Start sending
-                is_started = await self._start(self.input_stream_id)
+                is_started = await self._start()
                 if is_started:
-                    await self.send_data(self.input_stream_id)
-
-                # Stop Sending
+                    await self.send_data()
                 self.stop()
-                SendingProcess._logger.info(_MESSAGES_LIST["i000002"])
+                SendingProcess._logger.info("Execution completed.")
```

Comment:
a start to get rid of _MESSAGES_LIST["i000002"] like codes?

Reply:
yes